examples/01_basic/bolt_pretension.py

```diff
@@ -22,516 +22,871 @@
 
 """.. _ref_bolt_pretension:
 
-Bolt Pretension
+Bolt pretension
 ---------------
 
 This example demonstrates how to insert a Static Structural analysis
 into a new Mechanical session and execute a sequence of Python scripting
 commands that define and solve a bolt-pretension analysis.
 Scripts then evaluate the following results: deformation,
-equivalent stresses, contact, and bolt
+equivalent stresses, contact, and bolt.
 """
 
 # %%
-# Import necessary libraries
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Import the necessary libraries
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-import os
+from pathlib import Path
+import typing
 
 from PIL import Image
 from ansys.mechanical.core import App
 from ansys.mechanical.core.examples import delete_downloads, download_file
+import clr
 from matplotlib import image as mpimg
 from matplotlib import pyplot as plt
 from matplotlib.animation import FuncAnimation
 
 # %%
-# Embed mechanical and set global variables
+# Initialize the embedded application
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 app = App()
-app.update_globals(globals())
 print(app)
 
-cwd = os.path.join(os.getcwd(), "out")
-
-
-def display_image(image_name):
-    plt.figure(figsize=(16, 9))
-    plt.imshow(mpimg.imread(os.path.join(cwd, image_name)))
-    plt.xticks([])
-    plt.yticks([])
-    plt.axis("off")
-    plt.show()
+# Import the enums and global variables instead of using app.update_globals(globals())
+# or App(globals=globals())
+from ansys.mechanical.core.embedding.enum_importer import *
+from ansys.mechanical.core.embedding.transaction import Transaction
 
+clr.AddReference("System.Collections")
```

Comment:
Ideally these clr.AddReference calls should not be necessary - we should have the ability to import Quantity in one line, something like:
`from ansys.mechanical.core.embedding.importer import Quantity`